frontend/static/js/app.js

```diff
@@ -8,8 +8,8 @@ Vue.component("correction-view", {
       originalHadith: null,
       queueName: this.queue,
       diff: null,
-      hasCheckedComment: false,
-      reviewComment: '',
+      checked: false,

```

Comment:
checked is not descriptive enough. It is not clear what is checked. can we rename it to something more meaningful i.e. addComment or useComment?